### PR TITLE
Enable zuul integration jobs on molecule drivers

### PR DESCRIPTION
**Pull request by ssbarnea**
Adds default zuul projects from molecule drivers.

### CODE REVIEW

File: zuul.d/projects.yaml [hunk 1]
```diff
@@ -94,11 +99,30 @@
...
+- project:
+    name: github.com/ansible-community/molecule-hetznercloud
+    templates:
+      - system-required
+
 - project:
     name: github.com/ansible-community/molecule-libvirt
     templates:
       - system-required
-      - publish-to-pypi
```
Comment (pabelanger) [inline zuul.d/projects.yaml]: you want to stop publishing to pypi here?

Comment (ssbarnea) [reply zuul.d/projects.yaml]: No, but considering that publishing worked for molecule itself which does not have it, I assumed is not needed, being included in system ones. All repos should have it or not, this one was missing it.

**Comment by ssbarnea** [comment]
recheck

**Comment by ssbarnea** [comment]
recheck

**Comment by ssbarnea** [comment]
recheck

**Pull request merged**